app/models/user.rb

```diff
@@ -1,10 +1,15 @@
+# test
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] hi team, great job adding devise :clap: , I think you forgot just one thing, which to add the `:confirmable` module from devise to the user model, as mentioned in the requirements to **Ask for confirmation of email.** please make sure you have the :confirmable module from devise added here :+1: